modules/netinfo/netinfo.go

```diff
@@ -43,6 +43,16 @@ func (s State) Enabled() bool {
 	return s.State > netlink.NotPresent
 }
 
+// Unknown returns true if a network interface is in Unknown state.
+func (s State) Unknown() bool {
+	return s.State == netlink.Unknown
+}
+
+// Gone returns true if a network interface just disappeared.
+func (s State) Gone() bool {
+	return s.State == netlink.Gone
+}
```

Comment:
This should be inverted: "Present() { s.State != netlink.Gone }"

Reply:
Isn't `Present` than the same to `Enabled`?

Reply:
nvm.. I have updated the PR.

Reply:
Ah, you're right. "Present" is already overloaded. Gone is better to avoid confusion :)
Sorry for the flip-flip

Reply:
no problem, fixed.